.github/workflows/build.yml

```diff
@@ -20,6 +20,10 @@ jobs:
         uses: actions/setup-java@v1
         with:
           java-version: 11
```

Comment:
Can we remove all the references to previous JDK configs we don't need? I'd like to use the minimum version possible. So if we can use JDK 11, please keep JDK 11 and remove the versions we don't need 😃